strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/booters/StorageBooter.java

```diff
@@ -49,8 +46,11 @@
     @Inject
     private PropertiesBooter propertiesBooter;
 
-    private Path lockFile;
+    private ILock lock;
 
+//    TODO: Inject hazelconfiguration as dependency
```

Comment:
Is this `TODO` still necessary?

Reply:
So sorry that I left my TODOs after completing the work. I will fix it right away.